cns/cns_util.go

```diff
@@ -67,6 +67,13 @@ func dropUnknownCreateSpecElements(c *Client, createSpecList []cnstypes.CnsVolum
 				updatedEntityMetadata = append(updatedEntityMetadata, cnstypes.BaseCnsEntityMetadata(k8sEntityMetadata))
 			}
 			createSpec.Metadata.EntityMetadata = updatedEntityMetadata
+			createSpec.BackingObjectDetails.(*cnstypes.CnsBlockBackingDetails).BackingDiskUrlPath = ""
+			updatedcreateSpecList = append(updatedcreateSpecList, createSpec)
+		}
+		createSpecList = updatedcreateSpecList
+	} else if c.serviceClient.Version == ReleaseVSAN70 {
+		for _, createSpec := range createSpecList {
+			createSpec.BackingObjectDetails.(*cnstypes.CnsBlockBackingDetails).BackingDiskUrlPath = ""
```

Comment:
for calls which does not set `createSpec.BackingObjectDetails` we will get null pointer exception.

Reply:
Fixed this in  #1942